### ISSUE TITLE
Review annotations to steroid esterification and children

### ISSUE THREAD

**Issue by hdrabkin** [geneontology/go-annotation#3627]

**Comment by hdrabkin** [comment]
Interpro: 1 (InterPro:IPR030687)
BHF-UCL 13
MGI 1
TAIR 2
UniProt 11
ZFIN 1

**Comment by tberardini** [comment]
@ebakker2  Can you please review the TAIR annotations?  Thanks.

**Comment by sabrinatoro** [comment]
ZFIN annotation reviewed

**Comment by tberardini** [comment]
Reference ticket: https://github.com/geneontology/go-announcements/issues/194

I think the annotation review task is to reannotate to either a different process term and/or a molecular function term.

**Comment by ebakker2** [comment]
ok thanks!

**Comment by EricaBakker** [comment]
Updated one and obsoleted the other. Did not have edit access to spreadsheet to mark those changes.

**Comment by RLovering** [comment]
What about the regulation of this process? are these terms staying?

**Comment by pgaudet** [comment]
> any ideas how this happened?

No, I think you opened it... ? You were working on the corresponding ontology ticket, #19255

**Comment by sarach06** [comment]
Hi Pascale,
Thank you very much. The term has been removed from this entry; this will be visible in InterPro 91.0.
Best regards,
Sara
